lib/kernel/src/dist_util.erl

```diff
@@ -136,18 +136,25 @@ handshake_other_started(#hs_data{request_type=ReqType}=HSData0) ->
     ?debug({"MD5 connection from ~p (V~p)~n",
 	    [Node, HSData#hs_data.other_version]}),
     mark_pending(HSData),
-    {MyCookie,HisCookie} = get_cookies(Node),
+    Cookie = auth:get_cookie(Node),
```

Comment:
Replacing get_cookies(Node) with auth:get_cookie(Node) looks correct but seems unnecessary.
It should be ok to just call make_sure_cookie_is_set(MyCookie, Node), no?

Reply:
The replacement basically removes is_atom(check) (that is done in get_cookie()). I agree that it should never really affect the behavior, but who knows. I think I would prefer to avoid the unnecessary change.